app/src/main/AndroidManifest.xml

```diff
@@ -72,6 +72,16 @@
 
         <category android:name="android.intent.category.LAUNCHER" />
       </intent-filter>
+
+      <intent-filter android:autoVerify="true">
+        <action android:name="android.intent.action.VIEW"/>
+        <category android:name="android.intent.category.DEFAULT"/>
+        <category android:name="android.intent.category.BROWSABLE"/>
+        <data
+          android:scheme="https"
+          android:host="groundforis.com"
```

Comment:
You can use `groundplatform.org` as the host for the main codebase. @rfontanarosa will override to `ground.openforis.org` in the Open Foris fork.